Kudu.Services/Diagnostics/ProfileManager.cs

```diff
@@ -0,0 +1,314 @@
+﻿using Kudu.Contracts.Tracing;
```

Comment:
@mathewc this class has dependencies on some executables that need to be running on server. I was gonna add more test scenarios in Antares hosting for these, in there we could create sites, start/stop profiling etc.